proto/src/determined/api/v1/trial.proto

```diff
@@ -756,7 +766,16 @@ message CompareTrialsRequest {
   Scale scale = 7;
   // x-axis selection. Default is in batch.
   XAxis x_axis = 8;
+  // metric ids for the query
+  repeated string metric_ids = 9;
```

Comment:
@ioga You commented that in our system we usually only use `Id` for numbers or UUIDs. When this ticket was written the plan was to make metricIds numbers at some points. However, I do not know where this stands now. ccing @hkang1 for input also concerning the future of metricsIds.

Reply:
I don't see a way to introduce numeric metricIds.

Reply:
What you have now with a prefix metric type + metric name seems acceptable to me. The ideal to be able to reference a metric with one "thing" (in this case a string of format `<type>.<name>`) is good enough of an improvement.

Reply:
Sounds good, I just wanted to confirm that we were officially ditching numeric `metricIds`!